packages/instrumentation-qdrant/tests/instrumentation.test.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Traceloop
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { context } from "@opentelemetry/api";
+import { AsyncHooksContextManager } from "@opentelemetry/context-async-hooks";
+import { QdrantInstrumentation } from "../src/instrumentation";
+import {
+  BasicTracerProvider,
+  InMemorySpanExporter,
+  SimpleSpanProcessor,
+} from "@opentelemetry/sdk-trace-base";
+import * as qdrant from "@qdrant/js-client-rest";
+
+const COLLECTION_NAME = "some_collection";
+
+const memoryExporter = new InMemorySpanExporter();
+
+describe("Test Qdrant instrumentation", function () {
+  const provider = new BasicTracerProvider();
+  let instrumentation: QdrantInstrumentation;
+  let contextManager: AsyncHooksContextManager;
+  const qdrantClient: qdrant.QdrantClient = new qdrant.QdrantClient({ url: 'http://127.0.0.1:6333' });
```

Comment:
@Anush008 I think the issue is that you're initializing qdrant too early. See OpenAI example:
https://github.com/traceloop/openllmetry-js/blob/f30c012bf1762d88d2637ab5e0379ab0b0df9a72/packages/instrumentation-openai/test/instrumentation.test.ts#L64

Reply:
Unfortunately, that didn't work. Still at https://github.com/traceloop/openllmetry-js/pull/278#discussion_r1611341021.